src/app/mork/mail.py

```diff
@@ -0,0 +1,85 @@
+"""Email related functions."""
+
+import smtplib
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from logging import getLogger
+from smtplib import SMTPException
+
+from jinja2 import Environment, FileSystemLoader
+
+from mork.conf import settings
+from mork.exceptions import EmailSendError
+from mork.templatetags.extra_tags import Base64StaticTag
+
+logger = getLogger(__name__)
+
+
+def render_template(template: str, context) -> str:
+    """Render a Jinja template into HTML."""
+    template_env = Environment(
+        loader=FileSystemLoader(
+            [
+                settings.ROOT_PATH / "templates/html",
+                settings.ROOT_PATH / "templates/text",
+            ]
+        ),
+        autoescape=True,
+        extensions=[Base64StaticTag],
+    )
+    template = template_env.get_template(template)
+    return template.render(**context)
+
+
+def send_email(email_address: str, username: str):
+    """Initialize connection to SMTP and send a warning email."""
+    template_vars = {
+        "title": "Votre compte va être supprimé dans 30 jours.",
+        "email": email_address,
+        "fullname": username,
+        "site": {
+            "name": settings.EMAIL_SITE_NAME,
+            "url": settings.EMAIL_SITE_BASE_URL,
+        },
+    }
+    html = render_template(
+        "warning_email.html",
+        template_vars,
+    )
+
+    text = render_template(
+        "warning_email.txt",
+        template_vars,
+    )
+
+    # Create a multipart message (with MIME type multipart/alternative) and set headers
+    message = MIMEMultipart("alternative")
+    message["From"] = settings.EMAIL_FROM
+    message["To"] = email_address
+    message["Subject"] = "Your account will be closed soon"
```

Comment:
Do you forgot to make this message translatable or do you plan to internationalize this project later ?

Reply:
I'm planning to do it later, but I'm opening an issue to not forget about it (see X).
Changed the email subject to stick to a full french mail for now.